src/shipit_bot_uplift/shipit_bot_uplift/api.py

```diff
@@ -0,0 +1,113 @@
+import mohawk
+import requests
+import json
+import os
+from shipit_bot_uplift.helpers import ShipitJSONEncoder
+
+
+class ApiClient(object):
```

Comment:
Great idea ! We could use something like [bravado](https://github.com/Yelp/bravado)